prepackPy/prepackPy.py

```diff
@@ -12,63 +12,76 @@ def splitter(X, target_index, split_size, seed):
 Parameters
 ----------
 X: numpy array, pandas dataframe
-    the input dataset
+    an input dataset
 target_index: integer
-    the column index of target variable in X
+    a column index of target variable in X
 split_size: float
-    the proportion of the dataset to include in the test split
+    a proportion of the dataset to include in the test split
 seed: integer
-    the random state that will make code reproducible
+    a random state that will make code reproducible
 
 Returns
 -------
 X_train: numpy array
-    the splitted features for model training
+    a splitted features for model training
```

Comment:
maybe just "split features for model training"?

Reply:
OK, I will fix it.